src/member/apps.py

```diff
@@ -3,3 +3,9 @@
 
 class MemberConfig(AppConfig):
     name = 'member'
+
+    def ready(self):
+        # import signal handlers
+        # (every models imported inside handlers will be instantiated as
+        # soon as the registry is fully populated.)
+        import member.signals.handlers  # noqa
```

Comment:
@aaboffill  You can keep the function in `receivers.py`, and simply refer to that function in `ready` method.

The fact is I didn't understand this import at the first reading until noticing the side effect of the `@receiver` decorator that registers the function to Django's system. Semantically, calling `post_delete.connect()` says more clearly that we may have state changes on `post_delete` object.